packages/ecs-agent/0005-bottlerocket-fix-procfs-path-on-host.patch

```diff
@@ -0,0 +1,25 @@
+From aa1f344f6a13f1533a1eb93d7de419739bf3b8c3 Mon Sep 17 00:00:00 2001
+From: Samuel Karp <skarp@amazon.com>
+Date: Fri, 26 Mar 2021 17:48:28 -0700
+Subject: [PATCH 5/5] bottlerocket: fix procfs path on host
+
```

Comment:
This is because we're not running the agent in a container, right? Can you add a comment to this patch to explain why the path is fixed up this way?

Reply:
Yes, can add a comment.

Reply:
Done